### PR TITLE
SAK-94 Remove default auth override code

### PR DESCRIPTION
**Pull request by ern**
- Added unit test for SecurityService.securityCheck
- updated .gitignore with intellij exceludes

### CODE REVIEW

**Comment by kaltura-hooks** [comment]
Hi @ern,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

File: sakai.properties [hunk 1]
```diff
@@ -26,7 +26,9 @@
 
 # Kaltura RESTful API authorization override code
 # Allows access to /kaltura/user/{userId} endpoint without sending in persistent auth_code
-# Default: c48cb080-852b-11e4-80c2-0002a5d5c51b
+# If no override code is configured then this feature is disabled
+# If configured it should conform to the java.util.UUID version 4
+# see https://www.uuidgenerator.net/version4
 #kaltura.authorization.override.code=c48cb080-852b-11e4-80c2-0002a5d5c51b
```
Comment (gonenradai) [inline sakai.properties]: Don't you think this sample value also better be removed? to avoid cases where someone just uncomments the line?

**Comment by gonenradai** [comment]
@ern 
after merging SAK-98 into master, this PR now conflicts.
can you please fix the conflicts, so I can merge and we can continue with QA?

Thanks,
Gonen

**Comment by ern** [comment]
@gonenradai updated to resolve conflicts

**Pull request merged**